### ISSUE TITLE
Create LC qith example of lightning:dualActionList

### ISSUE THREAD

**Issue by rreboucas** [rreboucas/sparkle-dx#52]

**Comment by HoltMichael** [comment]
Couldn't find any reference to a lightning:dualActionList component?

**Comment by calvin-ou** [comment]
I looked into this as well and couldn't find any resources about the component. 
@rreboucas Are you referencing lightning:dualListbox instead?
https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/aura_compref_lightning_dualListbox.htm

**Comment by rreboucas** [comment]
@HoltMichael @calvin-ou That's right I was referring to [lightning:dualListbox](https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/aura_compref_lightning_dualListbox.htm). Tks for catching that!
